postgres/metadata.csv

```diff
@@ -84,6 +84,13 @@ postgresql.queries.blk_read_time,count,,nanosecond,,"Total time spent reading bl
 postgresql.queries.blk_write_time,count,,nanosecond,,"Total time spent writing blocks per query_signature, db, and user. (DBM only)",0,postgres, postgres queries block write time,
 postgresql.queries.duration.max,gauge,,nanosecond,,"The age of the longest running query per user, db and app. (DBM only)",0,postgres,postgres queries duration max,
 postgresql.queries.duration.sum,gauge,,nanosecond,,"The sum of the age of all running queries per user, db and app. (DBM only)",0,postgres,postgres queries duration sum,
+postgresql.slru.blks_zeroed,count,,block,,"Number of blocks zeroed during initializations of SLRU (simple least-recently-used) cache.",0,postgres,postgres slru zeroed,
+postgresql.slru.blks_hit,count,,block,,"Number of times disk blocks were found already in the SLRU (simple least-recently-used), so that a read was not necessary (this only includes hits in the SLRU, not the operating system's file system cache).",0,postgres,postgres slru hit,
+postgresql.slru.blks_read,count,,block,,"Number of disk blocks read for this SLRU (simple least-recently-used) cache.",0,postgres,postgres slru read,
+postgresql.slru.blks_written,count,,block,,"Number of disk blocks written for this SLRU (simple least-recently-used) cache.",0,postgres,postgres slru written,
+postgresql.slru.blks_exists,count,,block,,"Number of blocks checked for existence for this SLRU (simple least-recently-used) cache.",0,postgres,postgres slru exists,
+postgresql.slru.flushes,count,,,,"Number of flushes of dirty data for this SLRU (simple least-recently-used) cache.",0,postgres,postgres slru flushes,
+postgresql.slru.truncates,count,,,,"Number of truncates for this SLRU (simple least-recently-used) cache.",0,postgres,postgres slru truncates,
```

Comment:
While our other metrics don't do a great job of this, can you include more information about these metrics? I.e. what is each metric, what does its value indicate, is it good or bad if it's high or if it's zero, etc.? The documentation is an opportunity to educate our users about which metrics they should care about or not.

Reply:
I've tried to add more details on the doc but there's a 400 character limits which make it a bit hard to provide significant details.

Reply:
I was not aware we had such a limit. I asked the Agent integrations team why there is such a low limit.

Reply:
Here's a CI run that failed due to the 400 characters limit https://github.com/DataDog/integrations-core/actions/runs/3839186044/jobs/6536610056